mgmtfn/dockplugin/dockplugin.go

```diff
@@ -39,8 +39,12 @@ import (
 )
 
 const pluginPath = "/run/docker/plugins"
+
+//const pluginPath = "/usr/share/docker/plugins"
 const driverName = "netplugin"
 
+const useUnixSocket = true
```

Comment:
is there some reason we need to keep the old code path if this is always going to be true?